hwdb/60-evdev.hwdb

```diff
@@ -345,6 +345,13 @@ evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnHP:pnHPSpectreNotebook*
  EVDEV_ABS_35=1205:5691:47
  EVDEV_ABS_36=1083:4808:65
 
+# HP Envy x360
+evdev:name:SynPS/2 Synaptics TouchPad:dmi:bvnInsyde:bvrF.16:bd09/21/2018:svnHP:pnHPENVYx360Convertible15m-cn0xxx:pvrType1ProductConfigId:rvnHP:rn8483:rvr70.30:cvnHP:ct31:cvrChassisVersion:*
```

Comment:
please shorten this to the bits that matter - if it's too narrow it won't apply to all the devices it should. In this case, a good match is probably
```
evdev:name:SynPS/2 Synaptics TouchPad:*svnHP:pnHPENVYx360Convertible15m-cn0xxx*
```

Reply:
Thanks, done